shell/platform/darwin/ios/framework/Source/FlutterEngineGroup.mm

```diff
@@ -79,7 +64,7 @@ - (FlutterEngine*)makeEngineWithOptions:(nullable FlutterEngineGroupOptions*)opt
                              initialRoute:initialRoute
                            entrypointArgs:entrypointArgs];
   }
-  [_engines addObject:[NSValue valueWithPointer:engine]];
+  [self.engines addObject:[NSValue valueWithPointer:(__bridge void*)engine]];
```

Comment:
Could you put a TODO with an issue on these bridges so that we're tracking that we need to improve this, given that it's a bad pattern for ARC code?

Reply:
Done.